FlowCal/excel_ui.py

```diff
@@ -407,6 +446,15 @@ def process_samples_table(samples_table,
         in `samples_table` requires transformation to MEF, a key: value
         pair must exist in mef_transform_fxns, with the key being equal to
         the contents of field "Beads ID".
+    beads_table : DataFrame
+        Table specifying beads samples used to generate
+        `mef_transform_fxns`. This is used to check if a beads sample was
+        taken at the same acquisition settings as a sample to be
+        transformed to MEF. For any beads sample and channel for which a
+        MEF transformation function has been generated, the following
+        fields should be populated: ``<channel> Amp. Type`` and
+        ``<channel> Detector Volt``. If `beads_table` is not specified, no
+        checking will be performed.
```

Comment:
agree that function should still work if `beads_table` is not specified. Might add `optional` next to the parameter type to be consistent with other optional parameters.